### PR TITLE
feat(web): Add Plausible to the website

### PR DESCRIPTION
**Pull request by ERosendo**
This PR adds the Plausible's snippet to the `base` template of the project. This PR fixes https://github.com/freelawproject/bigcases2/issues/32

### CODE REVIEW

**Comment by ERosendo** [comment]
Sure!

**Comment by ERosendo** [comment]
@mlissner I've removed the tag that creates the random int

File: bc/assets/templates/base.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% load static tailwind_tags %}
+{% load static %}{% load tailwind_tags %}
```
Comment (mlissner) [inline bc/assets/templates/base.html:1]: I'm pretty sure you can do this in newer versions of Django:

```suggestion
{% load static tailwind_tags %}
```

**Review by mlissner — CHANGES_REQUESTED** [review]
One suggestion, please. Go ahead and merge once you've looked into it, and we can verify that it's working by visiting plausible.com.

File: bc/assets/templates/base.html (hunk 1 above)
Comment (ERosendo) [reply bc/assets/templates/base.html]: You're right. We can do that with the `load` tag

**Review by mlissner — APPROVED** [review]

**Pull request merged**